lib/y2logsstep.pm

```diff
@@ -84,6 +84,7 @@ sub workaround_dependency_issues {
             sleep 2;
         }
     }
+    return 1;
```

Comment:
can you comment on this change? What does it change/fix?

Reply:
if the function detect the dependency issue returns 1, so we can know that at least there was a conflict. Otherwise returns undef (that is was originally was returning).

Reply:
let's hope for the best then :)